src/utils/formValidations.ts

```diff
@@ -0,0 +1,5 @@
+type TranslateFunction = (key: string) => string;
```

Comment:
this is unnecessary, you can use the exported type `TFunction` from `react-i18next` or `i18next`